### PR TITLE
Add testing instructions to README.md

### LINKED ISSUE #1893: Add testing instructions to README.md
In the past, the question of how to test a PR came up several times. To enable contributors to test pull requests, I propose adding clear testing instructions to https://github.com/Automattic/themes/blob/master/README.md.

### PR DESCRIPTION
**Pull request by nielslange**
Fixes #1893

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,21 @@ Found a bug you can fix? Fantastic! Patches are always welcome. Here are a coupl
 * Reference any existing issues that relate to your PR. This allows everyone to easily see all related discussions.
 
 By contributing code to our free themes, you grant its use under the [GNU General Public License v2 (or later)](LICENSE).
+
+## Testing pull requests
+
+### Using GitHub 
+
+1. Clone repository locally `git clone git@github.com:Automattic/themes.git`
+2. Look up featured branch of pull request, e.g. `update/#1889`
```
Comment (kjellr) [inline README.md:32]:
```suggestion
2. [Identify the branch](https://cloudup.com/cAy1j0D8tvH) of the pull request, e.g. `update/#1889` 
```

^ Links to to an image showing where to find this.

File: README.md [hunk 2]
```diff
@@ -23,3 +23,21 @@ Found a bug you can fix? Fantastic! Patches are always welcome. Here are a coupl
...
+### Using GitHub 
+
+1. Clone repository locally `git clone git@github.com:Automattic/themes.git`
+2. Look up featured branch of pull request, e.g. `update/#1889` 
+3. Check out featured branch of pull request, e.g. `git checkout update/#1889`
+4. Symlink or copy affected theme OR zip affected theme and import into WordPress site
+
+### Manually download
+
+1. Look up featured branch of pull request, e.g. `update/#1889`
```
Comment (kjellr) [inline README.md:38]:
```suggestion
1. [Click on the branch](https://cloudup.com/cAy1j0D8tvH) of the pull request, e.g. `update/#1889` 
```

File: README.md [hunk 3]
```diff
@@ -23,3 +23,21 @@ Found a bug you can fix? Fantastic! Patches are always welcome. Here are a coupl
...
+
+1. Clone repository locally `git clone git@github.com:Automattic/themes.git`
+2. Look up featured branch of pull request, e.g. `update/#1889` 
+3. Check out featured branch of pull request, e.g. `git checkout update/#1889`
+4. Symlink or copy affected theme OR zip affected theme and import into WordPress site
+
+### Manually download
+
+1. Look up featured branch of pull request, e.g. `update/#1889` 
+2. Download featured branch of pull request, e.g. `https://github.com/Automattic/themes/archive/update/%231889.zip`
```
Comment (kjellr) [inline README.md:39]:
```suggestion
2. Change the `/tree/` part of the branch's URL to `archive`, and add a `.zip` to the end to download a zip of the branch. e.g. `https://github.com/Automattic/themes/tree/update/%231889` would become `https://github.com/Automattic/themes/archive/update/%231889.zip`
```

^ Just makes things a little more clear.

**Comment by nielslange** [comment]
> Added just a few small suggestions, but other than that I think this is a great starting point!

Thanks for your suggestions, @kjellr! They're all great and I approved all of them.

**Comment by kjellr** [comment]
Excellent, thanks @nielslange!

**Pull request merged**